app/scripts/lib/auth-errors.js

```diff
@@ -145,6 +145,36 @@ define(function (require, exports, module) {
       errno: 135,
       message: t('Unable to deliver email')
     },
+    // Secondary Email errors
+    EMAIL_EXISTS: {
+      errno: 136,
+      message: t('This email was already verified by another user')
+    },
+    EMAIL_PRIMARY_EXISTS: {
+      errno: 139,
+      message: t('Secondary email must be different than your account email')
+    },
+    EMAIL_VERIFIED_PRIMARY_EXISTS: {
+      errno: 140,
+      message: t('Account already exists')
```

Comment:
Should this and the next error have different strings so we can differentiate in Sentry?